README.md

```diff
@@ -89,6 +89,20 @@ If you're running Robbie on a supercomputing cluster (HPC), you should use the r
 
 Additional configuration files are stored in the `./config` directory and may be useful templates for your work.
 
+## Visualisation
+
+Firstly, build the Docker image located in the robbie_viewer_server directory:
+
+```
+./build_docker.sh
+```
+
+once this is complete, run the viewer in the main Nextflow directory via:
+
+```
+run_robbie_viewer.sh
```

Comment:
If you agree with my other comment about adding a directory argument, you should add how to use it here